src/main/java/edu/kis/vh/nursery/base_data_types/IntLinkedList.java

```diff
@@ -1,10 +1,10 @@
-package edu.kis.vh.nursery.list;
+package edu.kis.vh.nursery.base_data_types;
 
-public class IntLinkedList {
+public class IntLinkedList implements IntStack {
 
 	Node last;
```

Comment:
Should be private. 
Lack of encapsulation in Node class (Task 3 in "accessory i hermetyzacja")